frame/atomic-swap/src/lib.rs

```diff
@@ -50,6 +50,7 @@ use frame_support::{
 	weights::Weight,
 	RuntimeDebugNoBound,
 };
+use frame_system::pallet_prelude::BlockNumberFor;
```

Comment:
I would make this be the standard: 

All pallets must contain these lines: 

```
use frame_system::pallet_prelude::*;
use frame_support::pallet_prelude::*;

// needed to make things work in construct_runtime
pub use pallet::*;

#[frame_support::pallet]
pub mod pallet {
	use super::*;
}
```

This can probably simplify a lot of imports.

Reply:
(story for another day, of course)